apps/web/app/(landing)/careers/page.tsx

```diff
@@ -0,0 +1,81 @@
+import { Border } from "@/components/landing/border";
+import { Button } from "@/components/landing/button";
+import { Container } from "@/components/landing/container";
+import { FadeIn } from "@/components/landing/fade-in";
+import { PageIntro } from "@/components/landing/page-intro";
+import { allJobs } from "contentlayer/generated";
+import Link from "next/link";
+
+export const metadata = {
+  title: "Jobs | Unkey",
+  description: "Stay up-to-date with the latest updates and changes to Unkey",
+  openGraph: {
+    title: "Changelog | Unkey",
+    description: "Stay up-to-date with the latest updates and changes to Unkey",
+    url: "https://unkey.dev/changelog",
+    siteName: "unkey.dev",
+    images: [
+      {
+        url: "https://unkey.dev/og/changelog",
+        width: 1200,
+        height: 675,
+      },
+    ],
+  },
+  twitter: {
+    title: "Changelog | Unkey",
+    card: "summary_large_image",
+  },
+  icons: {
+    shortcut: "/unkey.png",
+  },
+};
```

Comment:
seems you forgot to update the metadata to the job

Reply:
oops